boards/cats_rev1Pro/src/cli/cli_commands.c

```diff
@@ -33,7 +33,7 @@
 #include <string.h>
 
 /** CLI command function declarations **/
-
+static void cli_cmd_bl(const char *cmd_name, char *args);
```

Comment:
move this declaration to be together with _reboot() and _save(). They are organized by group